From 74e8b27091d7d0c714a97e6d561ed1670fa59109 Mon Sep 17 00:00:00 2001
From: Thomas Bogendoerfer <tsbogend@alpha.franken.de>
Date: Wed, 12 Apr 2023 15:21:38 +0200
Subject: [PATCH] Revert "MIPS: generic: Enable all CPUs supported by virt
 board in Kconfig"

This reverts commit de34007751aaba992373f2d659001a846aeb8811.

There are too many build errors, which needs to be sorted out first.

Signed-off-by: Thomas Bogendoerfer <tsbogend@alpha.franken.de>
---
 arch/mips/Kconfig | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index 47944eb5d7e3..7629e671381b 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -152,16 +152,12 @@ config MIPS_GENERIC_KERNEL
 	select PCI_DRIVERS_GENERIC
 	select SMP_UP if SMP
 	select SWAP_IO_SPACE
-	select SYS_HAS_CPU_CAVIUM_OCTEON
-	select SYS_HAS_CPU_LOONGSON2E
-	select SYS_HAS_CPU_LOONGSON2F
 	select SYS_HAS_CPU_MIPS32_R1
 	select SYS_HAS_CPU_MIPS32_R2
 	select SYS_HAS_CPU_MIPS32_R6
 	select SYS_HAS_CPU_MIPS64_R1
 	select SYS_HAS_CPU_MIPS64_R2
 	select SYS_HAS_CPU_MIPS64_R6
-	select SYS_HAS_CPU_R4X00
 	select SYS_SUPPORTS_32BIT_KERNEL
 	select SYS_SUPPORTS_64BIT_KERNEL
 	select SYS_SUPPORTS_BIG_ENDIAN
-- 
2.39.1

